src/nnue/evaluate_nnue.cpp

```diff
@@ -126,10 +127,28 @@ namespace Eval::NNUE {
   // Evaluation function. Perform differential calculation.
   Value evaluate(const Position& pos) {
 
-    alignas(kCacheLineSize) TransformedFeatureType
-        transformed_features[FeatureTransformer::kBufferSize];
+    // We manually align the arrays on the stack because with gcc < 9.3
+    // overaligning stack variables with alignas() doesn't work correctly.
+
+    constexpr uint64_t alignment = kCacheLineSize;
```

Comment:
do we need the additional variable?

Reply:
no, but I think it's cleaner that way as it's used many times in the scope and `kCacheLineSize` only vaguely tells what it's about.